src/components/Plyrue.vue

```diff
@@ -31,6 +31,7 @@ export default {
   mounted() {
     const { $el, options, emitPlayerEvent } = this;
     this.player = new Plyr($el.firstChild, options);
+    this.player.poster = this.options.poster;
```

Comment:
Let's use `poster` prop. 
```suggestion
if(this.poster) this.player.poster = this.options.poster;
```